src/tasks/installers/installer.ts

```diff
@@ -89,7 +89,7 @@ export class InstallerTasks {
 
     // let task: Listr.ListrTask
 
-    if (flags.installer === 'operator') {
+    if (flags.installer === 'operator' || flags.installer === 'olm') {
```

Comment:
Sorry, but we have flag 'flags.installer === 'olm'' in the line 103. And with current change will be printed `'🏃‍  Running the Eclipse Che operator'`

Reply:
Right. Thank you